smartsim/_core/entrypoints/colocated.py

```diff
@@ -264,52 +280,62 @@ def cleanup() -> None:
             os.remove(LOCK.lock_file)
 
 
+def register_signal_handlers() -> None:
+    for sig in SIGNALS:
+        signal.signal(sig, handle_signal)
```

Comment:
Another signal handling change that looks like it snuck in from a different PR

Reply:
I can't recall the exact lint msg, but it was likely "too many locals" or something similar. I pulled the smallest amount out that I could that made sense to pass the test.